### PR TITLE
Update APC_costs.qmd with details on APC extrapolation

### PR DESCRIPTION
**Pull request by Enro**
Added some details on extrapolating unknown APCs, in past and future data.

### CODE REVIEW

**Comment by vtraag** [comment]
Thank you for this! Very much appreciated. I will take a closer look this Tuesday.

**Comment by vtraag** [comment]
Given that @igrypari and @PetrosStav were involved in the writing, it might also be good to get their input.

File: indicator_templates/quarto/1_open_science/APC_costs.qmd [hunk 1]
```diff
@@ -163,17 +163,29 @@ Via: <https://direct.mit.edu/qss/article/1/1/6/15582/Article-processing-charges-
...
 
-<https://research-and-innovation.ec.europa.eu/knowledge-publications-tools-and-data/publications/all-publications/monitoring-open-access-policy-horizon-2020_en>
+The French Open Science Monitor (*BSO*) also uses several extrapolations when the APC for a DOI is not available in OpenAPC (Bracco et al., 2022). They are listed here in order of preference (only when one computation is not possible is the following used):
+- average APC for the same journal and the same year in OpenAPC (provided n>= 10)
+- average APC for the same publisher in OpenAPC, for the same year as the article (provided n>= 10)
+- average APC for the same journal in OpenAPC, for all years available
+- average APC for the same publisher in OpenAPC, for all years available
+- APC for the journal in DOAJ
+
+To estimate a prospective amount of APC, Blanchard et al. (2022) used a mixed-effects model of the evolution of the APC  of an article as a function of the year and the publisher tier. In this mixed-effect model, the APC varies, and it is between publisher tiers that the variation is the largest.
```
[the remark this answers is not in the document]
Comment (Enro) [reply indicator_templates/quarto/1_open_science/APC_costs.qmd:180]: Sure, you are a better judge of this than I am!

File: indicator_templates/quarto/1_open_science/APC_costs.qmd [hunk 2]
```diff
@@ -163,17 +163,29 @@ Via: <https://direct.mit.edu/qss/article/1/1/6/15582/Article-processing-charges-
 -   Hybrid vs. Gold OA
 -   SNIP
 
-### Notes
+### Estimating unknown APCs
 
-An APC extrapolation exercise was conducted for the purposes of the following report.
+An APC extrapolation exercise was conducted for the purpose of the EC study (European Commission, 2021). The authors defined groupings, and imputed the average APC of the group to each publication in the group for which the APC is unknown. The groupings were based on the following variables, as per the correlates above:
```
[the remark this answers is not in the document]
Comment (Enro) [reply indicator_templates/quarto/1_open_science/APC_costs.qmd:168]: Makes sense, thank you.

**Comment by vtraag** [comment]
OK, @igrypari let me know via mail that there are no further comments. Let me clean it up a bit according to my comments above, and then I'll merge... Thanks!

**Comment by vtraag** [comment]
Thanks again @Enro! Feel free to open any other PR if you have anything else in the future!

**Pull request merged**